pkg/controller/operator.go

```
@@ -126,6 +127,37 @@ func (s *Operator) Run(ctx context.Context, controller *controllercmd.Controller
+	}
+	go kubeInf.Start(ctx.Done())
+	go configMapObserver.Run(ctx, 1)
+
+	// secretConfigObserver synthesizes all config into the status reporter controller
+	secretConfigObserver := configobserver.New(s.Controller, kubeClient)
+	go secretConfigObserver.Start(ctx)
+
+	configAggregator := configobserver.NewConfigAggregator(secretConfigObserver, configMapObserver)
+	go configAggregator.Listen(ctx)
+
+	// Create channel for update notifications
```

golangci-lint (2.11.4)

[error] todoCommentWithoutDetail: may want to add detail/assignee to this TODO/FIXME/BUG comment

(gocritic)